install.sh

```diff
@@ -3,8 +3,13 @@
 set -e
 
 osName=$(uname -s)
-DOWNLOAD_URL=$(curl --silent "https://api.github.com/repos/datreeio/datree/releases/latest" | grep -o "browser_download_url.*\_${osName}_x86_64.zip")
 
+osArchitecture=$(uname -m)
+if [[ osArchitecture == *'aarch'* || osArchitecture == *'arm'* ]]; then
+	osArchitecture = 'arm64'
```

Comment:
This if statement is not working well since you are using a string `osArchitecture` and not the actual variable `$osArchitecture `
also, you shouldn't have spaces in the assignment of `osArchitecture` to `arm64`

Reply:
Thanks for pointing out the issues. I have addressed review comment.